install/console-installer.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CONSOLE_INSTALL_PATH="$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)"
+RESOURCE_PATH=${CONSOLE_INSTALL_PATH}/resources
+
+export NAMESPACE="${1}"
+export CLUSTER_DOMAIN="${2}"
+
+source ${CONSOLE_INSTALL_PATH}/_common.sh
+
+PROVIDED_APIS="$(kubectl get operatorgroup -n ${NAMESPACE} -o json | jq -r '.items[].metadata.annotations["olm.providedAPIs"]' | tr '[:upper:]' '[:lower:]')"
+
+if ! kubectl get crd kafkas.kafka.strimzi.io ; then
+    echo -e "${ERROR} Strimzi Kafka Operator custom resource(s) not found"
+    display_suggested_subscription "strimzi-kafka-operator" "strimzi-kafka-operator|amq-streams"
+    exit 1
+fi
+
+### Prometheus
+kubectl apply -n ${NAMESPACE} -f ${RESOURCE_PATH}/prometheus/console-prometheus-server.clusterrole.yaml
+kubectl apply -n ${NAMESPACE} -f ${RESOURCE_PATH}/prometheus/console-prometheus-server.serviceaccount.yaml
+yq '.subjects[0].namespace = strenv(NAMESPACE)' ${RESOURCE_PATH}/prometheus/console-prometheus-server.clusterrolebinding.yaml | kubectl apply -n ${NAMESPACE} -f -
+kubectl apply -n ${NAMESPACE} -f ${RESOURCE_PATH}/prometheus/kafka-resources.podmonitor.yaml
+kubectl apply -n ${NAMESPACE} -f ${RESOURCE_PATH}/prometheus/kubernetes-scrape-configs.secret.yaml
+kubectl apply -n ${NAMESPACE} -f ${RESOURCE_PATH}/prometheus/console-prometheus.prometheus.yaml
```

Comment:
Is this required even if we have already deployed Prometheus?  Also, if we do already have Prometheus installed, can we just copy the `PodMonitor` and  `Secret` with the scrape configs or is it more complicated than that?